ctlcmd/motioncmd.py

```diff
@@ -261,27 +261,25 @@ def meas_str(v, u):
     self.printmsg(f'Best y:' + meas_str(fitval[2], np.sqrt(fitcovar[2][2])))
     self.printmsg(f'Fit  z:' + meas_str(fitval[3], np.sqrt(fitcovar[3][3])))
 
-    detid = str(args.detid)  ## Ensuring string convention in using this
-    ## Generating calibration det id if using det coordinates
-    if not detid in self.board.dets() and int(detid) < 0:
-      self.board.add_calib_det(detid, args.mode, args.channel)
+    detid = int(args.detid)  ## Ensuring int convention in using this
 
     ## Saving session information
+    coords = [
+      fitval[1],
+      np.sqrt(fitcovar[1][1]), 
+      fitval[2],
+      np.sqrt(fitcovar[1][1])
+    ]
     if not self.board.lumi_coord_hasz(detid, args.scanz) or args.overwrite:
-      self.board.add_lumi_coord(detid, args.scanz, [
-          fitval[1],
-          np.sqrt(fitcovar[1][1]), fitval[2],
-          np.sqrt(fitcovar[1][1])
-      ])
+      self.board.add_lumi_coord(detid, args.scanz, coords)
+      self.conditions.update_gantry_and_sipm_conditions(self.classname, detid, args.scanz)
```

Comment:
`Error: 'halign' object has no attribute 'conditions'` 

So you will need to expose the conditions to the `commandcmd` classes here: https://github.com/UMDCMS/SiPMCalibControl/blob/master/ctlcmd/cmdbase.py#L439